src/main/java/net/dv8tion/jda/core/audio/AudioWebSocket.java

```diff
@@ -521,7 +517,7 @@ private void setupKeepAlive(final int keepAliveInterval)
 
         try
         {
-            keepAlivePool.scheduleAtFixedRate(keepAliveRunnable, 0, keepAliveInterval, TimeUnit.MILLISECONDS);
+            keepAliveHandle = keepAlivePool.scheduleAtFixedRate(keepAliveRunnable, 0, keepAliveInterval, TimeUnit.MILLISECONDS);
```

Comment:
Hmmmm just realized this actually overwrites when it's called a second time.

How do you want me to fix it instead?